internal/event/event_observation.go

```diff
@@ -107,8 +107,11 @@ func (o *observation) ComposeFrom(events []*eventlogger.Event) (eventlogger.Even
 			if _, ok := payload[DetailsField]; !ok {
 				payload[DetailsField] = []gated.EventPayloadDetails{}
 			}
+			if g.Op == "" {
```

Comment:
@jimlambrt I think this check is never going to happen because in `newObservation` there is a validation check for an empty `op` string however, since `ComposeFrom` calls from the filter, Do you  think is there any situation that could lead to an empty `op`?

Reply:
I agree, it should be unreachable unless newObservation changes.

Reply:
Sure then I will remove it

Reply:
@jimlambrt should be this for 0.15.x release?